### PR TITLE
Added economic data and basic scatterplot implementation

### PR DESCRIPTION
**Pull request by mackliet**

### CODE REVIEW

File: js/Data/MigrationPatterns.ts [hunk 1]
```diff
@@ -55,7 +55,6 @@ export enum MigrationNodeId {
     'West Virginia',
     Wisconsin,
     Wyoming,
-    'Puerto Rico',
```
Comment (reillykw) [inline js/Data/MigrationPatterns.ts:58]: Why was this removed?  I know Puerto Rico wasn't in the map, but there's a way to get it there

File: js/Data/State_indicators.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
+/**
+ * Enumerator representing all 50 States and DC
+ */
+export enum State_id
```
Comment (reillykw) [inline js/Data/State_indicators.ts]: Can the MigrationNodeId type in MigrationsPatterns.ts be used here instead of creating a new type?

File: js/Views/IView.ts [hunk 3]
```diff
@@ -4,6 +4,4 @@ import {MigrationData} from "../Data/MigrationPatterns";
 export interface IView {
 
     readonly curYear: number;
-    readonly currentData: MigrationData;
```
Comment (reillykw) [inline js/Views/IView.ts:7]: I see you've dropped this type and you're using a StateIndicators type.  Maybe the interface here is overkill for the project and we can just create some global state context that each view reads and updates on when a year is changed.

**Review by reillykw — CHANGES_REQUESTED** [review]
This looks good, nice work.  The naming styles are a bit wonky, I think standard for projects like this is camel and pascal case, as opposed to pythons camel case (which is fine for the python scripts). 

The only thing I want to sync on before merging is the MigrationNodeId and State_id, because they look the same it seems like that could be imported from some DataUtils.ts file and renamed as 'RegionEnum' or something.

File: js/Views/Scatterplot.ts [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+        this.svgDims = svgDims;
+        this.padding = 100;
+        this.transition_time = 800;
+        
+        this.create_dropdowns();
+        this.create_scales();
+        this.update_plot();
+    };
+
+    static indicator_to_name(indicator)
```
Comment (reillykw) [inline js/Views/Scatterplot.ts:51]: This is a global problem I've been having, actually.  Might be good to have a standard way to do this.

File: js/Data/MigrationPatterns.ts (hunk 1 above)
Comment (mackliet) [reply js/Data/MigrationPatterns.ts:58]: There was no economic data for Puerto Rico or DC, and since they don't really show up on the map anyway I figured we could just take them off. I thought it might be odd in the table too if they don't have any data other than immigration data

File: js/Views/Scatterplot.ts (hunk 4 above)
Comment (mackliet) [reply js/Views/Scatterplot.ts:51]: I like the idea of a function like this, but there's more than one place we're doing it, a common place for that stuff would be good
